### ISSUE TITLE
Communicate with Sensor

### ISSUE THREAD

**Issue by kpakert** [kpakert/embedded-linux-final-project-kpakert#6]
Set up and deploy an application image that is able to communicate with the sensor.

DoD: ~~Able to communicate with the sensor and see output to the terminal via ssh~~ Determine how I plan to communicated with the sensor, and how I plan to make the application image.

Blockers: ~~Wiring and I2C enable~~ Unblocked

**Comment by kpakert** [comment]
Due to scope and time in sprint I have lowered the scope of this story, it was open ended and is requiring more work than expected. I have determined I plan to leverage the circuitpython library. This will involve properly adding a bitbake layer for the required modules, possibly require a custom recipe.

Additionally I will be able to leverage my python MQTT experience from the Embedded UI course in order to bring my stretch goal in scope for sprint 3.
